contracts/staking/GoodFundManager.sol

```diff
@@ -66,12 +66,12 @@ contract GoodFundManager is DAOContract {
     //Structure that hold reward information and if its blacklicksted or not for particular staking Contract
     struct Reward{
         uint32 blockReward; //in G$
-        uint32 blockStart;
-        uint32 blockEnd;
-        bool isBlackListed;
+        uint32 blockStart; // # of the start block to distribute rewards
+        uint32 blockEnd; // # of the end block to distribute rewards
+        bool isBlackListed; // If staking contract is blacklisted or not
```

Comment:
we can use uint64 here as it will still all be packed into single 256bits storage slot